src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftBuildVerifier.java

```diff
@@ -139,16 +80,11 @@ public String getDisplayName() {
             return DISPLAY_NAME;
         }
         
-        public long getWait() {
-        	return wait;
-        }
-
         @Override
         public boolean configure(StaplerRequest req, JSONObject formData) throws FormException {
             // To persist global configuration information,
             // pull info from formData, set appropriate instance field (which should have a getter), and call save().
-        	wait = formData.getLong("wait");
-        	GlobalConfig.setBuildVerifyWait(wait);
+        	GlobalConfig.setDefaultBuildVerifyWait(formData.getLong("wait"));
```

Comment:
Sure -- it was purely semantic. GlobalConfig serves as a default iff it is not specified in the individual build steps.